packages/components/src/Flex/Flex.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { Flex } from ".";
+
+afterEach(cleanup);
+
+it("renders the flexible container", () => {
```

Comment:
Wrap all of these test in a describe please! Makes it easier for our friends to see who these tests belong to at a glance when it runs with the others